bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/BigQueryClient.java

```diff
@@ -760,6 +772,16 @@ public void loadDataIntoTable(
     }
   }
 
+  public void loadDataIntoTable(
```

Comment:
Please check if this method is used. It was reference twice in the code, and both use the longer signature